packages/core/src/transport/eventBridge.spec.ts

```diff
@@ -45,11 +47,45 @@ describe('getEventBridge', () => {
     deleteEventBridgeStub()
   })
 
-  it('event bridge should serialize sent events', () => {
+  it('should serialize sent events without view', () => {
     const eventBridge = getEventBridge()!
 
     eventBridge.send('view', { foo: 'bar' })
 
     expect(sendSpy).toHaveBeenCalledOnceWith('{"eventType":"view","event":{"foo":"bar"}}')
   })
+
+  it('should serialize sent events with view', () => {
+    const eventBridge = getEventBridge()!
+
+    eventBridge.send('view', { foo: 'bar' }, '123')
+
+    expect(sendSpy).toHaveBeenCalledOnceWith('{"eventType":"view","event":{"foo":"bar"},"view":{"id":"123"}}')
+  })
+})
+
+describe('event bridge getPrivacyLevel', () => {
+  let eventBridgeStub: DatadogEventBridge
+  const bridgePrivacyLevel = DefaultPrivacyLevel.MASK
+
+  beforeEach(() => {
+    eventBridgeStub = initEventBridgeStub({ privacyLevel: bridgePrivacyLevel })
+  })
+
+  afterEach(() => {
+    deleteEventBridgeStub()
+  })
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**♻️ opportunity:** use registerCleanupTask for the event bridge stub‏